app/helpers/posts_helper.rb

```diff
@@ -1,2 +1,23 @@
 module PostsHelper
+  def render_text_section(post)
+    content_tag(:div, class: 'text') do
+      concat content_tag(:p, "Comments: #{post.CommentsCounter.nil? ? 0 : post.CommentsCounter}",
+                         class: 'comment-text')
+      concat content_tag(:p, "Likes: #{post.LikesCounter.nil? ? 0 : post.LikesCounter}")
+    end
+  end
+
+  def render_comments_section(post)
+    content_tag(:ul, class: 'ul-comments-box') do
+      concat content_tag(:h4, 'Section for Comments:', class: 'section-comments')
+
+      if post.comments.blank?
+        concat content_tag(:li, 'no comments for the moment')
+      else
+        post.comments.each do |comment|
+          concat content_tag(:li, "Username: #{comment.Text.blank? ? 'no comments for the moment' : comment.Text}")
+        end
+      end
```

Comment:
- [ ] [Optional] [Highly recommended] You can use partials for repeatable parts of your views, instead of what you used here helpers see [this](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials) for more info.